src/frontends/pytorch/src/op/native_multi_head_attention.cpp

```diff
@@ -0,0 +1,195 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_native_multi_head_attention(const NodeContext& context) {
+    /* pytorch.org/cppdocs/api/function_namespaceat_1aa4f72ac82c15c7aeef274332b25a543b.html
```

Comment:
I don't think exact link is really needed here

Reply:
Removed!